src/common/database.c

```diff
@@ -2731,7 +2731,7 @@ void ask_for_upgrade(const gchar *dbname, const gboolean has_gui)
   char *label_text = g_markup_printf_escaped(_("the database schema has to be upgraded for\n"
                                                "\n"
                                                "<span style='italic'>%s</span>\n"
-                                               "\n"
+                                               "\nthis might take a very long time in case of a large database\n\n"
```

Comment:
I would remove the "very" as it is a bit frightening to me, saying it could takes hours or days. The upgrade is generally fast or done in "just" several minutes.